lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,69 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+                if ($('.accordion').length) {
+                    navigation.openAccordion();
+                }
+            },
+
+            openAccordion: function() {
+                navigation.checkForCurrent();
+                navigation.listenForClick();
+            },
+
+            checkForCurrent: function() {
+                var active = $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') ? $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') : 0,
```

Comment:
This line of code is too long. I think it would be better to have a function called getActiveIndex that spreads out the code over several lines and makes it easier to read.